modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/Fields.tsx

```diff
@@ -645,7 +688,13 @@ const Fields = ({
 
 		const responseJSON = await response.json();
 
-		const storedFDSFields = responseJSON?.items;
+		const storedFDSFields = responseJSON?.items.map((field: any) => {
```

Comment:
Need a loop over the items to find the cases where there are translations but none of them is for the current language (`label` property). In this case, select the default field name.
Otherwise we could end up with an empty string for the field label.